specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/MitreCoverage.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2022-01-01-preview",
+    "x-ms-code-generation-settings": {
+      "name": "SecurityInsightsClient"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/mitreCoverage": {
```

Comment:
>mitreCoverage [](http://example.com/codeflow?start=176&length=13)

Resource type names are plural: mitreCoverages

Reply:
i understand,
but coverage is a name that contains multiple coverage for products.
i can call it MitreCoverageRecords, but in my opion the MitreCoverage is a better name,
let me know what do you think @mentat9

Reply:
So is this resource type a singleton? If so, the path should be to a single named instance: .../Microsoft.SecurityInsights/mitreCoverages/default

Reply:
I see. You want the resource type itself to be a collection, but that's not how ARM models resources. You should name this resource as the plural of the individual resources returned in the list. Sounds like mitreCoverageRecords is the way to go here.

Reply:
changed to mitreCoverageRecords